TSG/LCM/Add-Node-Failed-at-SetTrustedHostsOnNewNode-Step.md

```diff
@@ -0,0 +1,165 @@
+# How to Resume Add Node operation on BrownField Environment failed at SetTrustedHostsOnNewNode Step
```

Comment:
what is defined here as brown field? Is the regular add node scenario after deploying 23H2 cluster?

Reply:
not 23H2, this is for environments upgraded from 22H2 to 23H2